packages/xo-web/src/common/copiable/index.js

```diff
@@ -18,7 +18,6 @@ const Copiable = ({ className, tagName = 'span', ...props }) =>
       className: classNames(styles.container, className),
     },
     props.children,
-    ' ',
```

Comment:
Why this change?

Reply:
This was an extra space added to the content, like on the screenshot above (the space between the UUID and the double quote):
![image](https://github.com/vatesfr/xen-orchestra/assets/66562640/ad6c8bf8-ae5a-4221-8fb1-7cd69aeff7f5)
I removed it for formatting/style purposes, after checking that this did not break the display in the other locations the component is used.